src/components/Wizard/wizard.scss

```diff
@@ -63,7 +62,8 @@
         background-color: bootstrap.$input-focus-bg;
         border-color: bootstrap.$input-focus-border-color;
         outline: 0;
-        box-shadow: bootstrap.$input-box-shadow, bootstrap.$input-focus-box-shadow;
+        box-shadow: bootstrap.$input-box-shadow,
```

Comment:
Can you undo the formatting changes to this file? There currently isnt any formatting enforced, so we should set that up in a separate change. I filed #36 for that.